django_celery_beat/models.py

```diff
@@ -122,8 +122,9 @@ def from_schedule(cls, schedule):
         except cls.DoesNotExist:
             return cls(**spec)
         except MultipleObjectsReturned:
-            cls.objects.filter(**spec).delete()
-            return cls(**spec)
+            # unique_together constraint should not permit reaching this code,
+            # but just in case, we return the first schedule found
+            return cls.objects.filter(**spec).first()
```

Comment:
In the interest of avoiding dead code, why not drop these lines altogether?

Reply:
Given the constraint is here from the very beginning (https://github.com/celery/django-celery-beat/commit/ec46367c38a63a01e3e9ea6992534bc914650746) we can indeed drop the MultipleObjectsReturned block